### DISCUSSION TITLE
Spec generation is useless

### DISCUSSION THREAD

**Discussion by alberto-bottarini** [tailflow/laravel-orion#270] [Bugs]
I chose Laravel Orion for its spec generation feature, but after some days of usage I will remove it as dependency, and I'm not really happy about it.
I took this decision mainly because spec generation simply does not work well. I found 2 missing feature that make this automatic generation unusable:
- custom Resource does not work, I continue see all the model property, even if they are hidden using Resource class (https://github.com/tailflow/laravel-orion/discussions/258)
- pagination disabled trait is ignored, spec continue to show meta and links property even if the api does not include them in response (https://github.com/tailflow/laravel-orion/issues/269)

Additionally I did not found any feasible workaround to fix these.

**Comment by patton-dev** [comment]
Did you find another generator that worked?

This project seems half abandoned..

**Reply by gkimpson** [reply]
Appears nobody seems to reply to messages :(
